src/core/Microsoft.PowerToys.Settings.UI/Views/ImageResizerPage.xaml

```diff
@@ -62,7 +62,10 @@
                       ItemsSource="{x:Bind ViewModel.Sizes, Mode=TwoWay}"
                       Padding="0"
                       IsEnabled="{x:Bind Mode=OneWay, Path=ViewModel.IsEnabled}"
-                      SelectionMode="None">
+                      SelectionMode="None"
+                      ScrollViewer.HorizontalScrollMode="Enabled"
+                      ScrollViewer.HorizontalScrollBarVisibility="Visible"
```

Comment:
Visibility should be set to Auto so that it appears only when the content gets cropped. Otherwise you can hover at the bottom of control and see the scrollbar even if it isnt active
![image](https://user-images.githubusercontent.com/32061677/94037128-e4388180-fd79-11ea-81b1-8af3231a48ae.png)